api/nodes/views.py

```diff
@@ -335,6 +335,16 @@ def perform_destroy(self, instance):
             raise ValidationError(err.message)
         node.save()
 
+    def get_renderer_context(self):
+        context = super(NodeDetail, self).get_renderer_context()
+        show_counts = is_truthy(self.request.query_params.get('meta[templated_by_count]', False))
```

Comment:
I think we could use the `related_counts` query parameter for this as well. I don't think we need a separate one for just this.